rx/core/operators/publish.py

```diff
@@ -5,7 +5,7 @@
 from rx.subjects import Subject
 
 
-def _publish(mapper=None) -> ConnectableObservable:
+def _publish(mapper=None) -> Callable[[Observable], ConnectableObservable]:
```

Comment:
Maybe add typing for the mapper argument as well? I'll approve anyway, just a thought.

Reply:
yes you're right, will fix it, thanks!